patches/api/0006-Adventure.patch

```diff
@@ -3608,7 +3608,10 @@ index 36b436e145a7215682b692a87ab894df25752c1d..c5e8bffef736365881953161ff812c98
      public PlayerLocaleChangeEvent(@NotNull Player who, @NotNull String locale) {
          super(who);
          this.locale = locale;
-+        this.adventure$locale = net.kyori.adventure.translation.Translator.parseLocale(locale); // Paper
++        // Paper start
++        java.util.Locale javaLocale = net.kyori.adventure.translation.Translator.parseLocale(locale);
++        this.adventure$locale = javaLocale != null ? javaLocale : java.util.Locale.US;
```

Comment:
Why don't you use `requireNonNullElse` here like at the other spots?

Reply:
I find requireNonNullElse to be rather unwieldy. It was used at the other location because otherwise 200+ patches would have their line numbers changed.

If the maintainers would prefer
`this.adventure$locale = java.util.Objects.requireNonNullElse(net.kyori.adventure.translation.Translator.parseLocale(locale), java.util.Locale.US);`
here then I'll make that change

Reply:
It's just weird in terms of consistency.